File: an2_sem4/MPP/Homework/sapt11/java/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/objectprotocol/UpdatedSpectacolResponse.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.objectprotocol;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.dto.SpectacolDTO;

public class UpdatedSpectacolResponse extends UpdateResponse{
    public UpdatedSpectacolResponse(){}

    SpectacolDTO spectacol;

    public SpectacolDTO getSpectacol() {
        return spectacol;
    }

    public UpdatedSpectacolResponse(SpectacolDTO spectacol) {
        this.spectacol = spectacol;
    }
}

File: an2_sem4/MPP/Homework/sapt11/cs/FestivalSellpoint/FestivalSellpoint.Network/ObjectProtocol/UpdatedSpectacolResponse.cs
using FestivalSellpoint.Network.DTO;
using System;

namespace FestivalSellpoint.Network.ObjectProtocol
{
    [Serializable]
    public class UpdatedSpectacolResponse : IResponse
    {
        public SpectacolDTO Spectacol { get; }
        public UpdatedSpectacolResponse(SpectacolDTO spectacol) => Spectacol = spectacol;
    }
}
